source/2020-09-25-the-ember-times-issue-164.md

```diff
@@ -34,14 +35,16 @@ The blog also discusses using [`keyForAttribute`](https://api.emberjs.com/ember-
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [ember-concurrency 2.0.0.beta-1 🎉](https://twitter.com/m4xm4n/status/1308945677800681473)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+There's a [new beta](https://github.com/machty/ember-concurrency/releases/tag/2.0.0-beta.1) of `ember-concurrency` released! Thanks to the hard work of [Max Fierke (@maxfierke)](https://github.com/maxfierke) 2.0.0.beta-1 introduces `@tracked` on Ember 3.16+, removes the ability to directly use the `{{aciton helper}}`, and removes the `get` and `set` compatibility methods from `Task`, `TaskGroup`, and `TaskInstance`.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+As Max mentions in his tweet, 2.x is largely compatible with 1.x, so for users that maintain addons that depend on `ember-concurrency`, ti would be very helpful if you can use a wider version range so that folks can try the 2.x betas, and upgrade when they're able. 
```

Comment:
```suggestion
As Max mentions in his tweet, 2.x is largely compatible with 1.x. For users that maintain addons that depend on `ember-concurrency`, it would be very helpful if you can use a wider version range so that some folks can try the 2.x betas and others can upgrade when they're able to. 
```